.changeset/warm-gifts-study.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-kafka': minor
+'@backstage/plugin-kafka-backend': minor
+---
+
+Created alpha version of Kafka plugin, supporting listing consumer group for a Backstage configured service.
```

Comment:
No need to add a changeset for new plugins! It'll just result in a second release with no updates straight after the initial one.